src/app/component/organisms/abstract-session-management/abstract-session-management.component.ts

```diff
@@ -0,0 +1,64 @@
+import {Component, OnInit} from '@angular/core';
+import {Question} from "../../../model/question/question.model";
+import {Participant} from "../../../model/participant/participant.model";
+import {ActivatedRoute, Router} from "@angular/router";
+import {MessageService, Message} from "primeng/api";
+import {SessionService} from "../../../service/sessionService/session.service";
+
+export interface IAbstractSessionManagementComponent{
+  navigateToLogin(): void;
+  startConnection(token: string): void;
+}
+
+
+@Component({ template: '' })
+export class AbstractSessionManagementComponent implements OnInit, IAbstractSessionManagementComponent {
+
+  sessionId: number|null = null;
+  questions: Question[] = [];
+  participants: Participant[] = [];
+
+  constructor(
+    protected  router: Router,
+    protected  route: ActivatedRoute,
+    protected  messageService: MessageService,
+    protected  sessionService: SessionService,
```

Comment:
there are two spaces between protected and variablename. Personally I would choose to only use one.

Reply:
Wird mit dem Ticket LFD-38 upgedatet.